molecule/default/converge.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Converge
+  hosts: all
+  become: true
+
+  vars:
+    - newrelic_license_key: 'oKG5x5DRIuxVeHvijFXZo2Bl0T7UZsN2hioWIdlp'
```

Comment:
Where is this key from? Need to make sure we're not leaking a legit key here.

Reply:
Not a legit one.